.travis.yml

```diff
@@ -4,44 +4,53 @@
 language: generic
 
 os: osx
-osx_image: beta-xcode6.1
+osx_image: xcode6.4
 
 env:
   matrix:
     
     - CONDA_PERL=5.20.3.1
+    - CONDA_PERL=5.22.0.1
+    - CONDA_PERL=5.22.2.1
```

Comment:
Since `perl` is only used in the testing stage (is it?) this will make sense to pin to latest.

@kynan WDYT?